WooCommerce/WooCommerceTests/Mocks/MockReceiptEligibilityUseCase.swift

```diff
@@ -0,0 +1,9 @@
+@testable import WooCommerce
+
+final class MockReceiptEligibilityUseCase: ReceiptEligibilityUseCaseProtocol {
+    var isEligibleForBackendReceipts: Bool = false
```

Comment:
Should this default to true? Optimistically, most stores should be eligible, right?

Reply:
Yes, that's a good point. I'll update it 👍